### PR TITLE
[dev] PBI AB#56716: [Tech Debt] Add MUI Timeline Components

### PR DESCRIPTION
**Pull request by cbrewer1689**

### CODE REVIEW

File: docs/src/app/navigationConstants.js [hunk 1]
```diff
@@ -392,6 +392,17 @@ export const navigationItems = [
                 path: 'inputs',
             },
             {
+                directory: 'lab',
+                label: 'Lab',
```
Comment (groberts314) [inline docs/src/app/navigationConstants.js:396]: Just curious: Why "Lab" as the category for the Timeline?

File: docs/src/lab/timeline/timeline.jsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                    >
+                        {description}
+                    </Typography>
+
+                    <Heading
+                        anchorLink="address"
+                        variant="h2"
+                    >
+                        Timeline
+                    </Heading>
```
Comment (groberts314) [inline docs/src/lab/timeline/timeline.jsx:53]: Why `anchorLink="address"`?

File: src/lab/timeline/timeline.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import MuiTimeline from '@material-ui/lab/Timeline';
+import React from 'react';
+
+/**
+ * The timeline displays a list of events in chronological order.
+ */
+
+function Timeline(props) {
```
Comment (groberts314) [inline src/lab/timeline/timeline.jsx:8]: I don't think we need empty line 7 there.  Usually a doc block comment like this goes right on top of the thing it pertains to.  But Not a Blocker and not a problem if we want to keep it.

**Review by groberts314 — APPROVED** [review]
Looks good.  A couple of questions and one NAB; no blockers.

File: docs/src/app/navigationConstants.js (hunk 1 above)
Comment (cbrewer1689) [reply docs/src/app/navigationConstants.js:396]: That is where MUI put's this component. It's there beta bucket to say, "hey, these components may have issues. Use at your own risk."

File: docs/src/lab/timeline/timeline.jsx (hunk 2 above)
Comment (cbrewer1689) [reply docs/src/lab/timeline/timeline.jsx]: fixed.

File: src/lab/timeline/timeline.jsx (hunk 3 above)
Comment (cbrewer1689) [reply src/lab/timeline/timeline.jsx]: fixed.

File: docs/src/lab/timeline/timeline.jsx (hunk 2 above)
Comment (groberts314) [reply docs/src/lab/timeline/timeline.jsx]: Resolving.

File: src/lab/timeline/timeline.jsx (hunk 3 above)
Comment (groberts314) [reply src/lab/timeline/timeline.jsx]: Resolving.

**Pull request merged**